layouts/partials/head.html

```diff
@@ -10,12 +10,16 @@
 
     <link rel="canonical" href="{{ .Permalink | relURL }}">
 
-    <link rel="stylesheet" href="{{ "css/global.css" | relURL }}">
+    <link  rel="stylesheet" href="{{ "css/global.css" | relURL }}">
 
     <link rel="stylesheet" href="{{ "css/custom.css" | relURL }}">
 
     <link rel="stylesheet" href="{{ "css/search.css" | relURL }}" />
 
+    <!-- dark theme -->
+    <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js"></script>
+    <script src="/js/dark.js"></script>
```

Comment:
please do not use `/js/dark.js` directly. it may cause error when the `baseUrl` is not set to `/`. you may need to use `${params.baseURL}js/dark.js`

Reply:
fixed in be364faba91ae0503aff6291692755873c62f517